src/base/QXmppJingleIq.h

```diff
@@ -296,13 +321,17 @@ class QXMPP_EXPORT QXmppJingleIq : public QXmppIq
     QString responder() const;
     void setResponder(const QString &responder);
 
-    // XEP-0167: Jingle RTP Sessions
-    bool ringing() const;
-    void setRinging(bool ringing);
-
     QString sid() const;
     void setSid(const QString &sid);
 
+    RtpSessionState rtpSessionState() const;
+    void setRtpSessionState(RtpSessionState rtpSessionState);
+
+#if QXMPP_DEPRECATED_SINCE(1, 5)
+    bool ringing() const;
+    void setRinging(bool ringing);
+#endif
```

Comment:
please also mark them as deprecated

Reply:
I took that from `QXmpp::Encryption`. Do we have to add `QT_DEPRECATED_X()` to `QXmpp::Encryption` as well?

Reply:
`QT_DEPRECATED_X()` does not work on constants, but there's prob also something for that, so would be good yes.